lib/client/api_test.go

```diff
@@ -709,3 +710,51 @@ func TestVirtualPathNames(t *testing.T) {
 		})
 	}
 }
+
+func Test_formatConnectToProxyErr(t *testing.T) {
```

Comment:
super nit:

```suggestion
func TestFormatConnectToProxyErr(t *testing.T) {
```

Since formatConnectToProxyErr is what's being tested.